### DISCUSSION TITLE
timestamps to dates filter in supabase-js

### DISCUSSION THREAD

**Discussion by emersonlaurentino** [supabase/supabase#11351] [Feature Requests]
I have a column of type `timestamptz` and I want to do a query filtering using a date.

So, I can do this in SQL like this:
```sql
select * from table_name where column_name::date = '2022-12-29'
```

But, I can't do a query on supabase-js, I would like to do something like this:

```js
supabase
  .from("table_name")
  .select("*")
  .eq("column_name::date", "2022-12-29");
```

Today the only way I found to do this is by creating a Postgres Function and calling then using `.rpc`.

Would it be possible to add functionality similar to this in subbase-js?

**Comment by GaryAustin1** [comment]
You might be able to use the full time with gte and lte.  ` .gte('col','2022-19-29 00:00:00').lte('col','2022-19-29 23:59:59')`
https://www.postgresql.org/docs/current/datatype-datetime.html
I've not tried it though.
There is also the possibility of using a computer column to convert the column it to a date format
https://postgrest.org/en/stable/api.html#computed-virtual-columns
